spynnaker/pyNN/models/neuron/structural_plasticity/synaptogenesis/formation/distance_dependent_formation.py

```diff
@@ -16,6 +16,11 @@
 import numpy
 from spinn_utilities.overrides import overrides
 from .abstract_formation import AbstractFormation
+from data_specification.enums.data_type import DataType
+from spinn_front_end_common.utilities.constants import BYTES_PER_WORD
+
+# Number of words in the parameter struct excluding VLAs
+_WORDS_IN_STRUCT = 6
```

Comment:
habit here is to add to the comment what those elements are. and usually its bytes and you do the mul here in the define and not on line 78

Reply:
OK, fixed